frontend/packages/dev-console/src/components/monitoring/overview/monitoring-overview-alerts-utils.ts

```diff
@@ -0,0 +1,20 @@
+import * as _ from 'lodash';
+import { AlertSeverity, alertSeverityOrder } from '@console/internal/reducers/monitoring';
+import { Alert } from '@console/internal/components/monitoring/types';
+
+export const sortMonitoringAlerts = (alerts: Alert[]): Alert[] =>
+  _.sortBy(alerts, alertSeverityOrder) as Alert[];
```

Comment:
Do we need `as Alert[]` here?

Reply:
Otherwise, it gives a type error saying
`'Alert | Rule' is not assignable to type 'Alert'`